test/suites/lib/tarantool_python_ci.lua

```diff
@@ -343,5 +343,5 @@ end
 
 -- }}}
 
-clean()
+-- clean()
```

Comment:
Let's remove the dead code.

Reply:
Fixed clean to work with 2.x. I decided not to change current behavior of clean execution since tarantool_python_ci.lua may be used in external projects like in https://github.com/tarantool/tarantool/issues/6965